tests/rptest/services/compatibility/compat_example.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import sys
+import time
+from ducktape.services.background_thread import BackgroundThreadService
+
+
+class CompatExample(BackgroundThreadService):
+    def __init__(self, context):
+        super(CompatExample, self).__init__(context, num_nodes=1)
+        self._cmds = None
```

Comment:
why does `cmd` execute what is in `cmds[0]` but kills `cmds[1]`. maybe a couple sentences about how to use this would be helpful.